kyuubi-server/src/main/scala/org/apache/kyuubi/parser/plan/command/ShowSessions.scala

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.parser.plan.command
+
+import scala.collection.mutable.ListBuffer
+
+import org.apache.kyuubi.operation.IterableFetchIterator
+import org.apache.kyuubi.parser.schema.{Column, Row, Schema}
+import org.apache.kyuubi.parser.types.StringType
+import org.apache.kyuubi.session.KyuubiSession
+
+/**
+ * A runnable node for showing kyuubi sessions.
+ *
+ * The syntax of using this command in SQL is:
+ * {{{
+ *   SHOW KYUUBI_SESSIONS;
+ * }}}
+ */
+case class ShowSessions() extends RunnableCommand {
```

Comment:
why not use the thrift types directly like the spark SQL meta operations

Reply:
it takes a while to remove type-mapping with spark ones, so I suggest we remove these and use thrift types directly

Reply:
The Kyuubi parser module has its own SQL capability, so I think it would be better to have its own schema system. In addition, the newly added kyuubi runnable node can conveniently implement its own node's output schema instead of constructing thrift types, which is somewhat troublesome. we only need to convert the output once.

Reply:
but you add them to the operation manager, which based on TRowSet

Reply:
users call a kyuubi command through the different kind of FEs, which call a unified BE, which then call the OperationManager for RunnableCommands. So in which cases the users will touch the schema classes you create?

Reply:
The schema classes are not touched; these are used internally by the kyuubi parser module

Reply:
I do quite understand why we need these, I only see an unnecessary wrap/unwrap. And introducing an brand new type system is completely out of this PR's scope

Reply:
my idea is that the new runnableCommand can easily implement its own schema and data output. If each runnableCommand constructs thrift's schema and rowSet, I don't think it is convenient

Reply:
You can create a PR(better KPIP) and dev discussion for adding a new type system, it's not a small work. 

I'm truly sorry for the inconvenient about this request will cost, but it's necessary as an incomplete type system will result in a nightmare. See the timestamps/dates/intervals in hive/spark SQL type system for an example

Reply:
Thanks kent, agree with you, i will move out the type system-related changes in this pr and the thrift type is used directly.

Reply:
Any updates

Reply:
Remove the newly introduced type system and use thrift's `TTypeId`